packages/input-datepicker/src/LionInputDatepicker.js

```diff
@@ -186,8 +186,8 @@ export class LionInputDatepicker extends LionInputDate {
   get delegations() {
     return {
       ...super.delegations,
-      properties: super.delegations.properties.filter(p => p !== 'disabled'),
-      attributes: super.delegations.attributes.filter(p => p !== 'disabled'),
+      properties: super.delegations.properties.filter(p => p !== 'disabled' && p !== 'readOnly'),
+      attributes: super.delegations.attributes.filter(p => p !== 'disabled' && p !== 'readonly'),
```

Comment:
you already removed `readOnly`/`readonly` from delegation in LionInput (which I still don't understand to be honest)

but if so, why do you need to make this exception `p !== 'readonly'` here?

Reply:
Good catch. Will remove that part